template/java8/model/src/main/java/com/openfaas/model/IResponse.java

```diff
@@ -7,7 +7,9 @@
 
 public interface IResponse {
     String getBody();
+    byte[] getBodyData();
     void setBody(String body);
+    void setBody(byte[] body);
```

Comment:
Given that we have getBody and getBodyData, wouldn't it be more consistent to have `setBodyData`?

Reply:
I agree getBodyData()/setBodyData() would be consistent. On the other hand, one method (name) with different data types for setting body data is also consistent. But I can change this.